SGTCalculator/SGTCalculator.cs

```diff
@@ -34,4 +34,16 @@ static void Main(string[] args)
             Console.WriteLine(result);
```

Comment:
You should at your code inside the scope
After the other people scope, otherwise it won;t compile
       }
    }
}